src/index.js

```diff
@@ -516,16 +515,5 @@ let load = base.loader({
 
 }, ['profile', 'process']);
 
-// If this file is executed launch component from first argument
-if (!module.parent) {
-  load(process.argv[2], {
```

Comment:
This was moved to root/index to simplify the call and separate the bin from the module.